tools/cmake/toolchains/GCC_ARM.cmake

```diff
@@ -44,43 +46,30 @@ function(mbed_set_toolchain_options target)
         "-g3"
     )
 
-    target_compile_options(${target}
-        INTERFACE
-            ${common_options}
-    )
+    target_compile_options(${target} INTERFACE ${common_options})
```

Comment:
What options is affecting this line and line above 32 ? is the list above formatted differently because it is over 80 characters long so would not fit on the same line?